lib/API/JSON/v1/SendingQueue.php

```diff
@@ -177,20 +184,18 @@ public function pause($data = []) {
       ? (int)$data['newsletter_id']
       : false
     );
-    $newsletter = Newsletter::findOne($newsletterId);
+    $newsletter = $this->newsletterRepository->findOneById($newsletterId);
 
-    if ($newsletter instanceof Newsletter) {
-      $queue = $newsletter->getQueue();
+    if ($newsletter instanceof NewsletterEntity) {
+      $queue = $newsletter->getLatestQueue();
```

Comment:
I noticed that with this change we also change the logic of how is the queue fetched. The old model was using `updated_at` column ([#](https://github.com/mailpoet/mailpoet/blob/21839fc30b3fcdc57c9140998b33c325ecd7d33d/lib/Tasks/Sending.php#L141)), but `getLatestQueue` uses [order of ids](https://github.com/mailpoet/mailpoet/blob/3c5a368c2c6205a5bdaed1c4e023f7fa9147aa49/lib/Entities/NewsletterEntity.php#L461-L464). This should have probably very similar results, but maybe we could switch `getLatestQueue` to use the same logic or maybe better introduce a new getter `getLastUpdatedQueue()` so that we don't introduce some unexpected behaviour.